src/config/config.rs

```diff
@@ -210,6 +214,18 @@ fn merge_unions(
   self_unions
 }
 
+pub async fn merge_links(mut self_links: Vec<Link>, other_links: Vec<Link>) -> anyhow::Result<Vec<Link>> {
+  // let runtime = tokio::runtime::Runtime::new()?;
+  for link in other_links {
+    // let recurse_links = runtime.block_on(async { Link::resolve_recurse(link).await })?;
+    let recurse_links = Link::resolve_recurse(link).await?;
```

Comment:
IMO, we should have one step to merge the configs and other step to actually load it.

here we should make only the sync operation of merging the data structure. the loading of it should be on other step.